misc/scripts/install_d3d12_sdk_windows.py

```diff
@@ -54,23 +48,6 @@
 if not os.path.exists(deps_folder):
     os.makedirs(deps_folder)
 
-# DirectX Shader Compiler
-print("\x1b[1m[1/4] DirectX Shader Compiler\x1b[0m")
-if os.path.isfile(dxc_archive):
-    os.remove(dxc_archive)
-print(f"Downloading DirectX Shader Compiler {dxc_filename} ...")
-urllib.request.urlretrieve(
-    f"https://github.com/microsoft/DirectXShaderCompiler/releases/download/{dxc_version}/{dxc_filename}",
-    dxc_archive,
-)
-if os.path.exists(dxc_folder):
-    print(f"Removing existing local DirectX Shader Compiler installation in {dxc_folder} ...")
-    shutil.rmtree(dxc_folder)
-print(f"Extracting DirectX Shader Compiler {dxc_filename} to {dxc_folder} ...")
-shutil.unpack_archive(dxc_archive, dxc_folder)
-os.remove(dxc_archive)
-print(f"DirectX Shader Compiler {dxc_filename} installed successfully.\n")
-
 # Mesa NIR
 print("\x1b[1m[2/4] Mesa NIR\x1b[0m")
```

Comment:
We now have an off by one error ;)
(should be [1/3] and the other ones similarly reordered)

Reply:
Fixed.